### PR TITLE
Add podman_container_exec module

### LINKED ISSUE #480: Add `podman_container_exec`  module in a similar way as docker
/kind feature

**Description**

Currently one of the most lacking features of the `containers.podman` collection is the ability to execute commands on running containers in an easy way, similarly to how the `community.docker.docker_container_exec` operates. While it seems to be possible to use the `containers.podman.podman` connection, this is in no way well documented and may not be adequate for all use cases. Add a `podman_container_exec` to fill the gap in this otherwise fantastic collection.

### PR DESCRIPTION
**Pull request by nishipy**
Fixes #480

### CODE REVIEW

**Comment by sshnaidm** [comment]
Thanks @nishipy ! Let's use this template for testing, so we can see the running tests in job: https://github.com/containers/ansible-podman-collections/pull/531

**Comment by nishipy** [comment]
Thank you @sshnaidm for the template! Let me have a look within the next few days.

File: .github/workflows/podman_container_exec.yml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+      fail-fast: false
+      matrix:
+        ansible-version:
+          - ansible<2.10
+          # - git+https://github.com/ansible/ansible.git@stable-2.11
+          - git+https://github.com/ansible/ansible.git@devel
+        os:
+          - ubuntu-22.04
+        python-version:
+          - 3.9
```
[the remark this answers is not in the document]
Comment (nishipy) [reply .github/workflows/podman_container_exec.yml]: thanks

**Comment by nishipy** [comment]
It works now

**Comment by sshnaidm** [comment]
Awesome! Thanks for such a great work!

**Comment by DamianBarabonkovQC** [comment]
Hi all. This module is exactly what I need. I'm wondering when it might be released in the latest ansible collections? And if there's a workaround in the time being. Thanks!

**Comment by nishipy** [comment]
@sshnaidm Do you have any plan on it?

**Comment by sshnaidm** [comment]
@nishipy I'll try to release it today

**Comment by DamianBarabonkovQC** [comment]
Thanks so much!

**Comment by nishipy** [comment]
Thank you @sshnaidm

**Pull request merged**